src/decorators/debounce.ts

```diff
@@ -1,6 +1,12 @@
-import lodashDebounce, {DebounceSettings} from 'lodash/debounce';
+import debounce from 'lodash-es/debounce';
 
-export default function debounce(wait = 500, options?: DebounceSettings) {
+export interface DebounceSettings {
```

Comment:
I can get it from `import {DebounceSettings} from 'lodash'`
But trying to figure out if it will include the entire lodash or not?